prov/verbs/src/verbs_cm.c

```diff
@@ -257,6 +267,9 @@ vrb_msg_ep_accept(struct fid_ep *ep, const void *param, size_t paramlen)
 				       struct vrb_connreq, handle);
 		free(connreq);
 	}
+	if (!ret && _ep->profile) {
+		vrb_prof_cntr_inc(_ep->profile, FI_VAR_CONN_ACCEPT);
+	}
```

Comment:
Our convention is not to use `{` and `}` around single line after `if` statement unless a matching `else` requires `{` `}`. Same for a few other instances.